views/home.ejs

```diff
@@ -25,6 +28,7 @@
     const logoutButton = document.getElementById('logout');
     const profileButton = document.getElementById('profile');
     const events = document.getElementById('events');
+    const Analytics= document.getElementById('dashboard');
 
     if (signUpButton) {
         signUpButton.addEventListener('click', () => {
```

Comment:
`click` is repeated multiple times. Can we extract this to a variable or map of all the events?